### PR TITLE
added call for scan

### PR DESCRIPTION
**Pull request by yzainee**

### CODE REVIEW

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -737,9 +738,23 @@ def post():
         github_url = request.form.get("github_url")
         ref = request.form.get('github_ref')
         user_email = request.headers.get('UserEmail')
+        scan_repo_url = request.headers.get('scanRepoUrl')
+        print('Email ' + user_email)
```
Comment (miteshvp) [inline bayesian/api_v1.py]: Use logger statements as in - https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/api_v1.py#L761

Comment (miteshvp) [reply bayesian/api_v1.py]: I would not even want to log this information in our logs. especially user_email

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -737,9 +738,23 @@ def post():
...
+        print('GitUrl ' + scan_repo_url)
         if not user_email:
             user_email = decoded.get('email', 'bayesian@redhat.com')
 
+        if scan_repo_url is not None and scan_repo_url is not "":
```
Comment (miteshvp) [inline bayesian/api_v1.py]: or use `if scan_repo_url`. It will work exactly as this if.

File: bayesian/api_v1.py [hunk 3]
```diff
@@ -737,9 +738,23 @@ def post():
...
+            try:
+                api_url = GEMINI_SERVER_URL
+                data = {'git-url': scan_repo_url,
+                        'email-ids': [user_email]}
```
Comment (miteshvp) [inline bayesian/api_v1.py]: do we want to send a list here. I guess there will always be a single email associated to the build?

File: bayesian/default_config.py [hunk 4]
```diff
@@ -49,3 +49,7 @@
 CORE_DEPENDENCIES_REPO_URL = os.getenv('CORE_DEPENDENCIES_REPO', 'https://github.com/'
                                                                  'fabric8-analytics/'
                                                                  'booster-core-dependencies')
+
+GEMINI_SERVER_URL = "http://{host}:{port}".format(
```
Comment (miteshvp) [inline bayesian/default_config.py]: You can make use of gremlin url from - https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/utils.py#L41

File: bayesian/api_v1.py [hunk 5]
```diff
@@ -45,6 +45,7 @@
 import os
 from f8a_worker.storages import AmazonS3
 from .generate_manifest import PomXMLTemplate
+from .default_config import GEMINI_SERVER_URL
```
Comment (miteshvp) [inline bayesian/api_v1.py]: Use gremlin url available in utils

File: bayesian/api_v1.py (hunk 3 above)
Comment (yzainee) [reply bayesian/api_v1.py]: @samuzzal-choudhury wants its in a list format

File: bayesian/default_config.py (hunk 4 above)
Comment (miteshvp) [reply bayesian/default_config.py]: I meant, you can also use GEMINI in utils.

**Pull request merged**